modules/document_repository/php/document_repository.class.inc

```diff
@@ -410,6 +427,19 @@ class Document_Repository extends \NDB_Menu_Filter
             $DisplayItem['File_size'] = $this->formatSize($item['File_size']);
             $DisplayItem['For_site']  = $site_names[$item['For_site']];
 
+            $DisplayItem['edit_file']   = $user->hasPermission(
+                'document_repository_delete'
+            ) == true ?
+                '<a href=\'#\' id=\''. $DisplayItem['record_id']
```

Comment:
I don't think our coding standards allow for injecting raw HTML into the page via PHP. This should be done in another file.